internal/services/authorization/role_assignment_marketplace_resource_test.go

```diff
@@ -4,6 +4,14 @@ import (
 	"context"
 	"fmt"
 	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/authorization"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/authorization/2022-04-01/roledefinitions"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/testclient"
 
```

Comment:
Is there a reason why you left and empty line between the imports?

Reply:
Forgot to update the import paths. Code updated.